src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/CoverageModelEMParams.java

```diff
@@ -0,0 +1,1179 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
+
+import org.broadinstitute.barclay.argparser.Advanced;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+import javax.annotation.Nonnull;
+
+/**
+ * This class represents the hyper-parameters and parameters for {@link CoverageModelEMAlgorithm}
+ *
+ * Works both as an argument collection as well as a getter/setter.
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class CoverageModelEMParams {
```

Comment:
I think this should have ArgumentCollection in the name ("Params" makes me think that it might store model parameters, and probably shouldn't have been shortened in the first place).  Furthermore, I would remove all of the setter methods, since I would think this class only needs to store the argument settings used throughout a single run.  If an argument needs to be "updated" on the fly, then it probably does not really belong in this argument collection.